spec/factories/generic_works.rb

```diff
@@ -2,13 +2,18 @@
   factory :generic_work, aliases: [:work, :public_work], class: GenericWork do
     transient do
       user { FactoryGirl.create(:user) }
+      dates_of_work { [DateOfWork.new(start: (1900 + rand(100)).to_s)] }
     end
 
     title ['Test title']
     visibility Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC
 
     after(:build) do |work, evaluator|
       work.apply_depositor_metadata(evaluator.user.user_key)
+
+      # unfortunately this does force saving the dates_of_work
```

Comment:
I don't think so? But maybe? Can always be done later if we need it? I don't anticipate needing it.